Buildings/Controls/OBC/ASHRAE/G36/Atomic/DamperValveReheatBox.mo

```diff
@@ -163,63 +169,71 @@ block DamperValveReheatBox
-  CDL.Continuous.Sources.Constant conZer4(k=0) "Constant zero"
+  CDL.Continuous.Sources.Constant conZer4(final k=0) "Constant zero"
     annotation (Placement(transformation(extent={{-60,-130},{-40,-110}})));
-  CDL.Continuous.Sources.Constant conZer5(k=0) "Constant zero"
+  CDL.Continuous.Sources.Constant conZer5(final k=0) "Constant zero"
     annotation (Placement(transformation(extent={{-80,80},{-60,100}})));
-  CDL.Continuous.Sources.Constant conZer6(k=0) "Constant zero"
+  CDL.Continuous.Sources.Constant conZer6(final k=0) "Constant zero"
     annotation (Placement(transformation(extent={{80,-330},{100,-310}})));
-  CDL.Continuous.Sources.Constant conOne(k=1) "Constant one"
+  CDL.Continuous.Sources.Constant conOne(final k=1) "Constant one"
     annotation (Placement(transformation(extent={{-220,280},{-200,300}})));
-  CDL.Continuous.Sources.Constant conOne1(k=1) "Constant real value"
+  CDL.Continuous.Sources.Constant conOne1(final k=1) "Constant real value"
     annotation (Placement(transformation(extent={{0,-130},{20,-110}})));
-  CDL.Continuous.Sources.Constant conOne2(k=1) "Constant real value"
+  CDL.Continuous.Sources.Constant conOne2(final k=1) "Constant real value"
     annotation (Placement(transformation(extent={{-180,-360},{-160,-340}})));
-  CDL.Continuous.Sources.Constant conHal(k=0.5) "Constant real value"
+  CDL.Continuous.Sources.Constant conHal(final k=0.5) "Constant real value"
     annotation (Placement(transformation(extent={{-200,-130},{-180,-110}})));
-  CDL.Continuous.Sources.Constant conHal1(k=0.5) "Constant real value"
+  CDL.Continuous.Sources.Constant conHal1(final k=0.5) "Constant real value"
     annotation (Placement(transformation(extent={{-260,-360},{-240,-340}})));
-  CDL.Continuous.Sources.Constant lowDisAirTem(k=273.15 + 10)
+  CDL.Continuous.Sources.Constant lowDisAirTem(final k=minDisTem)
     "Lowest allowed discharge air temperature"
     annotation (Placement(transformation(extent={{-260,120},{-240,140}})));
-  CDL.Continuous.AddParameter addPar(p=maxDTem, k=1)
+  CDL.Continuous.AddParameter addPar(final p=maxDTem, final k=1)
     "Maximum heating discharge temperature"
     annotation (Placement(transformation(extent={{-260,-90},{-240,-70}})));
-  CDL.Continuous.AddParameter addPar1(k=1, p=2.8)
+  CDL.Continuous.AddParameter addPar1(final k=1, final p=2.8)
     "Zone temperature pluzonSetpoints 2.8 degC"
     annotation (Placement(transformation(extent={{-260,-280},{-240,-260}})));
-  CDL.Continuous.Greater gre
-    "Check if discharge air temperature is greater than zone temperature plus 2.8 degC"
-    annotation (Placement(transformation(extent={{-60,-280},{-40,-260}})));
-  CDL.Continuous.Greater gre1
-    "Check if supply air temperature is greater than room temperature"
-    annotation (Placement(transformation(extent={{-60,180},{-40,200}})));
-  CDL.Continuous.GreaterThreshold greThr
+  CDL.Continuous.Hysteresis hys(final uLow=0, final uHigh=0.05)
```

Comment:
If the control signal is never below, say `1E-20`, then this would never switch as u is bigger than uLow=0.
Hence, you need to set uLow = 0.01 (or 0.005).
The same needs to be applied below for the other hysteresis.